src/DependencyInjection/Configuration.php

```diff
@@ -35,6 +35,7 @@ public function getConfigTreeBuilder()
                             ->thenInvalid('"dc-aware-round-robin" load balancing option require a "dc_options" entry in your configuration')
                         ->end()
                         ->children()
+                            ->booleanNode('persistent_session')->defaultValue(true)->end()
```

Comment:
Yes, i wrote this using singular because it's an option on a client but I will match the cassandra driver method as you suggest.